packages/auth/src/authClient.ts

```diff
@@ -112,6 +115,19 @@ const mapAuthClientNetlify = (client: NetlifyIdentity): AuthClient => {
   }
 }
 
+const mapAuthClientMagicLinks = (client: MagicLinks): AuthClient => {
+  return {
+    type: 'magic.link',
+    client,
+    login: async ({ email }) =>
+      await client.auth.loginWithMagicLink({ email: email }),
+    logout: () => client.user.logout(),
```

Comment:
With the types added the logout now gives type ever
```
Type 'Promise<boolean>' is not assignable to type 'void | Promise<void>'.
  Type 'Promise<boolean>' is not assignable to type 'Promise<void>'.
    Type 'boolean' is not assignable to type 'void'.ts(2322)
authClient.ts(18, 3): The expected type comes from the return type of this signature.
```
it is becuase logout() returns a bool type but redwood asks for void() what do you think the best way to correct this is @peterp?

Reply:
> it is becuase logout() returns a bool type but redwood asks for void()

While I'm not too familiar with Redwood myself, this appears to be the case. You could fix the type error by doing this:

```ts
    logout: async () => { await client.user.logout() },
```